cmd/kubeadm/app/images/images_others.go

```diff
@@ -0,0 +1,29 @@
+// +build !windows
```

Comment:
let's keep this file named `images_unix.go`, to match the pattern of `constants_*.go`